tyr/clusters/mongo.py

```diff
@@ -43,6 +44,8 @@ def __init__(self, group=None, server_type=None, instance_type=None,
         self.data_nodes = data_nodes
         self.mongodb_version = mongodb_version
         self.dns_zones = dns_zones
+        if subnet_ids:
+            self.subnet_ids= [subnet_id.strip() for subnet_id in subnet_ids.split(',')]
```

Comment:
So it looks like you're expecting the value of `subnet_ids` to be a string like `"subnet-abc,subnet_def,subnet_ghi"`, give or take some spaces. I think we should just expect an actual list `["subnet-abc", "subnet_def", "subnet_ghi"]`.